src/src/TemplateBundle/Controller/WebTestController.php

```diff
@@ -81,7 +82,7 @@ public function testEventLanguagesAction($eventTypeId, Request $request) {
         $definedEventTypeLanguages = array_map($getNamesFromLanguage, $eventType->getLanguages()->toArray());
 
         $eventId = $request->query->get('eventId');
-        if ($eventId != null) {
+        if (null != $eventId) {
```

Comment:
yoda coding style? :D

Reply:
tiez to neznasam, ale phpfixer.

Reply:
Je to cool podla mna :) myslim, good practise.